src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -399,7 +399,7 @@ public void setStyle(final String style)
     @Override
     public boolean hasCapability(@NotNull final Capability<?> capability, final EnumFacing facing)
     {
-        if (capability == CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
+        if (capability == CapabilityItemHandler.ITEM_HANDLER_CAPABILITY && facing == null)
```

Comment:
Why null?
This would result in interactions with other mods probably breaking.

I left the null out so things like hoppers and Refined Storage etc could connect with our buildings, including the warehouse for inventory management, i am pretty sure this would upset some people.

Reply:
I did that to optimize for us finding the capabilities (We get like 10 handlers for each provider).
I'll undo that then and try to optimize a bit farther back, I thought we don't need this.

Reply:
This optimization is useless…. We Group them in a set later on anyway.

Reply:
I fixed the optimization now, so we can recognize in the grouping process if two are equal.